eeg_flow/tasks/annotate_bad_channels_and_segments.py

```diff
@@ -62,13 +62,15 @@ def bridges_and_autobads(
 
     # lock the output derivative files
     derivatives = (
-        derivatives_folder / "plots" / f"{fname_stem}_step1b_bridges.svg",
-        derivatives_folder / f"{fname_stem}_step1b_with-bads_raw.fif",
+        derivatives_folder / "plots" / f"{fname_stem}_step2_bridges.svg",
+        derivatives_folder / f"{fname_stem}_step2_with-bads_raw.fif",
     )
     locks = lock_files(*derivatives, timeout=timeout)
     try:
-        if not overwrite or all(derivative.exists() for derivative in derivatives):
+        if all(derivative.exists() for derivative in derivatives):
             raise FileExistsError
+        if not overwrite:
+            print("overwrite set to False")
```

Comment:
@ArthurNguyen Can you explain why you did this, so that I can clean-it up/improve correctly?

Reply:
oops yes, the condition is just wrong.